src/game.hpp

```diff
@@ -17,18 +20,23 @@ class Game {
 
 	void tick(kvf::Seconds dt);
 	void render(le::Renderer& renderer) const;
+	void increase_score(std::size_t points);
+	[[nodiscard]] std::string get_score_string() const;
```

Comment:
This seems unimplemented, and even so it shouldn't be allocating a new string on every call.